eventstore/migrations/0040_create_base_objects.up.sql

```diff
@@ -1,3 +1,8 @@
+-- "migration" written before we had proper migration handling, hence the
+-- various attempts at mitigating previously existing objects
+
+BEGIN;
+
```

Comment:
Should we get rid of the DROP TABLE's as we now delete the contents of the table on startup?